### ISSUE TITLE
Could you share the pretrained model on miniImageNet?

### ISSUE THREAD

**Issue by yinguoweiOvO** [nayeemrizve/invariance-equivariance#4]
Hello! Thank you for sharing codes.

I've reproduced results for CIFAR-FS and FC100, but I failed when I tried to reproduce the paper accuracies on miniImageNet.
Could you share the pretrained model on miniImageNet and tieredImageNet ? or give some tips?
Thanks

Best,
YGW

**Comment by nayeemrizve** [comment]
Hello YGW,

Thank you for expressing interest in our work and I am extremely sorry for
the late response. I have been busy with work recently and forgot to
respond to this before. For now, can you please try a batch size of 56 for
the miniImageNet and tieredImageNet experiments? I will try to share the
pretrained weights that I used during the submission or retrain and share
the weights by end of next week.

**Comment by yinguoweiOvO** [comment]
> Hello YGW, Thank you for expressing interest in our work and I am extremely sorry for the late response. I have been busy with work recently and forgot to respond to this before. For now, can you please try a batch size of 56 for the miniImageNet and tieredImageNet experiments? I will try to share the pretrained weights that I used during the submission or retrain and share the weights by end of next week. Best Regards, Mamshad Nayeem Rizve

Thanks for your reply and I'm sorry for the late response.
I think batch size has an impact on results because of the contrastive learning part. I tried to set the batch size to 32 and have reproduced the result on miniImageNet. But a large batch size cannot be set due to equipment problems. Do you have any good suggestions on how to modify the batch size without affecting the results of the experiment？

Best wishes to you,
YGW

**Comment by nayeemrizve** [comment]
I haven't tweaked the batch size parameter for experiments. Because of the memory constraint, I mainly used a batch size of 52/56 for the miniImageNet and tieredImageNet experiments. Now I see that batch size has some impact on results because of the contrastive loss. I guess to make the performance less sensitive to the batch size parameter non-contrastive instance classification losses (BYOL, SimSiam etc) can be used. But since I haven't experimented with these losses I am not sure what will be the overall performance.

**Comment by zhouchunpong** [comment]
> Hello! Thank you for sharing codes.
> 
> I've reproduced results for CIFAR-FS and FC100, but I failed when I tried to reproduce the paper accuracies on miniImageNet. Could you share the pretrained model on miniImageNet and tieredImageNet ? or give some tips? Thanks
> 
> Best, YGW

Hello, I use the hyperparamenters in README.md and cannot reproduced results for CIFAR-FS, do you change the hyperparamenters in your reproduced results ?

Best and Thanks

**Comment by VVVVVVY** [comment]
> 

I didn't change the hyperparameters. Maybe you should set the batch size to 64 or evaluate the model multiple times.

**Comment by yinguoweiOvO** [comment]
> > Hello! Thank you for sharing codes.
> > I've reproduced results for CIFAR-FS and FC100, but I failed when I tried to reproduce the paper accuracies on miniImageNet. Could you share the pretrained model on miniImageNet and tieredImageNet ? or give some tips? Thanks
> > Best, YGW
> 
> Hello, I use the hyperparamenters in README.md and cannot reproduced results for CIFAR-FS, do you change the hyperparamenters in your reproduced results ?
> 
> Best and Thanks

I didn't change the hyperparameters. Maybe you should set the batch size to 64 or evaluate the model multiple times.

**Comment by zhouchunpong** [comment]
> > > Hello! Thank you for sharing codes.
> > > I've reproduced results for CIFAR-FS and FC100, but I failed when I tried to reproduce the paper accuracies on miniImageNet. Could you share the pretrained model on miniImageNet and tieredImageNet ? or give some tips? Thanks
> > > Best, YGW
> > 
> > 
> > Hello, I use the hyperparamenters in README.md and cannot reproduced results for CIFAR-FS, do you change the hyperparamenters in your reproduced results ?
> > Best and Thanks
> 
> I didn't change the hyperparameters. Maybe you should set the batch size to 64 or evaluate the model multiple times.
Got it and Thank you very much ！
Best Wishes ！
